lima-init.sh

```diff
@@ -55,6 +55,8 @@ flag {
 }
 EOF
 echo "#LIMA-END" >> /etc/fstab
+mkmntdirs
+mount -a
```

Comment:
These two lines prevented lima from starting for me, not until they were executed as part of provision scripts. I noticed that during my tests since the 9p PR got merged into Lima.

Does it work fine for RD?